scripts/stops.js

```diff
@@ -46,4 +46,4 @@ async function fetchStops() {
     return stops.filter(stop => stopIds.includes(stop.stopId));
 }
 
-module.exports = fetchStops;
+module.exports = { fetchStops, fetchStopIds };
```

Comment:
`fetchStops` is required in scripts/server.js as a default import, so this PR needs to update that as well so that it continues working.